newscoop/src/Newscoop/GimmeBundle/EventListener/PublicResourcesListener.php

```diff
@@ -30,7 +30,7 @@ public function onRequest(GetResponseEvent $event)
     {
         $request = $event->getRequest();
         $route = $request->attributes->get('_route');
-        if (strpos($route, 'newscoop_gimme_') === false) {
+        if (strpos($route, 'newscoop_gimme_') === false || $route == 'newscoop_get_img') {
```

Comment:
I thinked about this but so far i don't have idea how to group those routes/files etc.